OptionSelector/DetailTableViewController.m

```diff
@@ -0,0 +1,67 @@
+//
+//  SubTableViewController.m
+//  OptionSelector
+//
+//  Created by Brian Blanco on 8/14/15.
+//  Copyright © 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "DetailTableViewController.h"
+
+@interface DetailTableViewController ()
+
+@end
+
+@implementation DetailTableViewController
+
+- (void)viewDidLoad {
```

Comment:
Style: If you have a method implementation that only calls `super`, then you can remove it altogether to keep your .m files slimmer and more readable.